.github/workflows/ci.yml

```diff
@@ -35,7 +36,7 @@ jobs:
           - ubuntu2204
           - centos7
         version:
-          - 8.0.0.3971
+          - 8.0.0.4020
```

Comment:
@timothylytle As the tested version will be updated frequently, it is better to load it from Github Secrets here

Reply:
@genben This has been implemented and tested, I also created a document that details how to modify

Reply:
Thank you @timothylytle